app/jobs/start_sync_job.rb

```diff
@@ -1,12 +1,20 @@
 class StartSyncJob < ApplicationJob
   attr_accessor :sync, :part_size
 
-  def perform(sync_id, part_size = OrganizationExportJob::PART_SIZE)
+  PART_SIZE = OrganizationExportJob::PART_SIZE
```

Comment:
One option would be to define it an env var. Batches are configured for performance, right? So unless we're adjusting the batch size at runtime, why define it in the implementation?